src/admin/admin.component.tsx

```diff
@@ -0,0 +1,176 @@
+              navigateHomeAriaLabel={'navigate to admin page'}
+            />
+            <NavigateNext
+              fontSize="medium"
+              sx={{ color: 'text.secondary', margin: 1 }}
+            />
+          </div>
+        </Grid>
+      </Grid>
+      {adminPageName === null && (
+        <Grid container flexDirection={'column'}>
+          <Grid item container xs={12} overflow={'auto'}>
+            <Grid item key={0} xs={12} sm={6}>
+              <Button
+                component={Link}
+                to={'units'}
+                fullWidth
+                sx={{
+                  display: 'flex',
+                  width: '100%',
+                  textDecoration: 'none',
+                  color: 'inherit',
+                  position: 'relative', // Make the parent container relative
+                }}
+              >
+                <Card
+                  sx={{
+                    padding: '8px',
+                    width: '100%',
+                    display: 'flex',
+                    height: '100px', // Set a fixed height for all cards
+                  }}
+                >
+                  <CardContent
+                    sx={{
+                      width: '100%',
+                      display: 'flex',
+                      justifyContent: 'center',
+                      alignItems: 'center',
+                      minWidth: 0,
+                    }}
+                  >
+                    <Grid>
+                      <Grid position={'relative'}>
+                        <Typography>Units</Typography>
+                      </Grid>
+                    </Grid>
+                  </CardContent>
+                </Card>
+              </Button>
+            </Grid>
+            <Grid item key={1} xs={12} sm={6}>
+              <Button
+                component={Link}
+                to={'usage-status'}
+                fullWidth
+                sx={{
+                  display: 'flex',
+                  width: '100%',
+                  textDecoration: 'none',
+                  color: 'inherit',
+                  position: 'relative', // Make the parent container relative
+                }}
+              >
+                <Card
+                  sx={{
+                    padding: '8px',
+                    width: '100%',
+                    display: 'flex',
+                    height: '100px', // Set a fixed height for all cards
+                  }}
+                >
+                  <CardContent
+                    sx={{
+                      width: '100%',
+                      display: 'flex',
+                      justifyContent: 'center',
+                      alignItems: 'center',
+                      minWidth: 0,
+                    }}
+                  >
+                    <Grid>
+                      <Grid position={'relative'}>
+                        <Typography>Usage Status</Typography>
```

Comment:
```suggestion
                        <Typography>Usage Statuses</Typography>
```
Could the names be changed to `Usage statuses` and `usage-statuses` to match units please.